jpype/_core.py

```diff
@@ -96,6 +96,10 @@ def decorate(func):
 
 
 def isJVMStarted():
+    """ This method is horribly named.  It should be named isJVMRunning as
```

Comment:
This can be a comment (e.g. TODO), but not a docstring IMO. It is confusing for the user.

Reply:
Any recommendations for a docstring as it doesnt do what its name implies?  This has been bothering me for a while.  We have three states being described by one bool flag: jvm not started, jvm running, jvm shutdown.